estate/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': "Real Estate",
```

Comment:
```suggestion
    'name': 'Real Estate',
```
Please ensure to have either ' ' or " " for all the occurrences in the file.

Reply:
changes done.